app/models/budget.rb

```diff
@@ -0,0 +1,12 @@
+class Budget < ApplicationRecord
+  belongs_to :user
+
+  validates :amount, presence: true, numericality: {greater_than: 0}
+  validates :category, presence: true
+  validates :month, presence: true
+  validates :flow, presence: true
+
+  def self.categories
```

Comment:
Ah I was wondering what the best way to get the user created categories would be. This looks simple and effective

Reply:
we should probably implement a case-insensitive approach and then have FE display as they wish